app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+
+
+from app.errors import NotVaccinatedError, \
+    NotWearingMaskError, OutdatedVaccineError
```

Comment:
It’s better to use brackets
`from app.errors import (
    NotVaccinatedError,
    NotWearingMaskError,
    OutdatedVaccineError
    ) `